specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
Please check with @TFR258 to see if we want to name this v3.2-preview.1 to anticipate the possibility of preview.2.

Reply:
Thanks! Have checked with Thomas, change the name as v3.2-preview.2.